packages/web-components/src/tree-item/tree-item.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { expect, test } from '../../test/playwright/index.js';
+import type { TreeItem } from './tree-item.js';
+
+test.describe('Tree Item', () => {
+  test.use({
+    tagName: 'fluent-tree-item',
+  });
+
+  test('should work with basic rendering', async ({ fastPage }) => {
+    const { element } = fastPage;
+    await fastPage.setTemplate({
+      innerHTML: `Item 1`,
+    });
+    await expect(element).toHaveText('Item 1');
+  });
+
+  test('should work with basic rendering - nested', async ({ fastPage }) => {
+    const { element } = fastPage;
+    await fastPage.setTemplate({
+      innerHTML: /* html */ `
+        <fluent-tree-item>Nested Item A</fluent-tree-item>
+      `,
+    });
+
+    const nestedItem = element.nth(0).locator('fluent-tree-item');
+    await expect(nestedItem).toHaveCount(1);
+    await expect(nestedItem).toHaveText('Nested Item A');
+  });
+
+  test('should work with expanded attribute', async ({ fastPage }) => {
+    const { element } = fastPage;
+    await fastPage.setTemplate({
+      innerHTML: /* html */ `
+        Item 1
+        <fluent-tree-item>Nested Item A</fluent-tree-item>
+      `,
+    });
+
+    await expect(element.nth(0)).not.toHaveAttribute('expanded');
+
+    const nestedItems = element.locator('fluent-tree-item');
+    await expect(nestedItems).toBeHidden();
+
+    await element.nth(0).evaluate((node: TreeItem) => {
+      node.expanded = true;
+    });
+    await expect(element.nth(0)).toHaveAttribute('expanded');
+    await expect(nestedItems).toBeVisible();
+  });
+
+  test('should work with selected attribute', async ({ fastPage }) => {
+    const { element } = fastPage;
+    await fastPage.setTemplate({
+      innerHTML: `Item 1`,
+    });
+
+    expect(await element.getAttribute('selected')).toBeNull();
+    // selected
+    await fastPage.setTemplate({
+      attributes: { selected: true },
+      innerHTML: `Item 1`,
+    });
+    expect(await element.getAttribute('selected')).not.toBeNull();
+  });
```

Comment:
Using `setTemplate()` twice in a fixture will cause the entire body to be emptied and rewritten with the new content. This can sometimes cause the page to be put in a dirty state, where the workings of the previous assertion affect subsequent assertions. It's safer to either change the `selected` property directly with `element.evaluate()` (which indicates that this modification is intentional), or split this into two separate tests.